packages/core/integration-tests/test/html.js

```diff
@@ -1539,16 +1539,16 @@ describe('html', function() {
     });
 
     let html = await outputFS.readFile(path.join(distDir, 'a.html'), 'utf8');
-    assert.equal(html.match(/<script/g).length, 3);
+    assert.equal(html.match(/<script/g).length, 2);
```

Comment:
The contents of the additional bundle before was empty (aside from the sourcemap comment). Looks like it was a case of this issue: a shared bundle was never properly removed from the graph.